design-tokens/tokens/layer3/base.json

```diff
@@ -84,6 +84,22 @@
           }
         }
       }
+    },
+    "offset": {
+      "dropdown": {
+        "submenu": {
+          "panel": {
+            "vertical": {
+              "value": "-4px",
```

Comment:
As I was saying in slack, I don't think this value should be -4px, that won't be usable on figma...